backend/.env.example

```
@@ -1,5 +1 @@
-MEMPOOL_URL=https://mempool0713bb23.devnet-annapurna.stratabtc.org/
-BLOCKSCOUT_URL=https://blockscoutb86fae58ae.devnet-annapurna.stratabtc.org/
-STRATA_DOCS="https://docs.stratabtc.com"
-STRATA_BLOG="https://www.stratabtc.org/blog"
-STRATA_FULLNODE="http://fnclient675f9eff3a682b8c0ea7423.devnet-annapurna.stratabtc.org"
+STRATA_FULLNODE="https://strataseq.temp6-testnet1-staging.stratabtc.org/"
```

This is .env.example but still i think everything goes through blutgang
```suggestion
STRATA_FULLNODE="https://rpc.testnet-staging.stratabtc.org/"
```